### PR TITLE
feat: improve keywords in builder to allow to edit

### PR DESCRIPTION
**Pull request by ReidyT**
This PR solve #84.
image

### CODE REVIEW

File: src/utils/keywords.ts [hunk 1]
```diff
@@ -11,3 +13,40 @@ export const replaceWordCaseInsensitive = (
   const regex = new RegExp(wordToReplace, 'gi');
   return text.replace(regex, replacement);
 };
+
+export const isKeywordsEquals = (
```
Comment (pyphilia) [inline src/utils/keywords.ts:17]:
```suggestion
export const areKeywordsEqual = (
```

File: src/components/views/admin/KeywordsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            </Typography>
+            {t(TEXT_ANALYSIS.KEYWORDS_NOT_IN_TEXT_TOOLTIP_MSG, {
+              keyword: row.word,
+            })}
+          </>
+        }
+      >
+        <WarningIcon
+          data-cy={buildKeywordNotExistWarningCy(row.rowId)}
+          sx={{ marginLeft: '5px', color: '#FFCC00' }}
```
Comment (pyphilia) [inline src/components/views/admin/KeywordsTable.tsx:54]: Does that work?
```suggestion
          sx={{ ml: 1, color: '#FFCC00' }}
```

File: src/components/views/admin/KeywordsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  [`& .${tooltipClasses.tooltip}`]: {
+    backgroundColor: '#f5f5f9',
+    color: 'rgba(0, 0, 0, 0.87)',
+    maxWidth: 220,
+    fontSize: theme.typography.pxToRem(12),
+    border: '1px solid #dadde9',
+  },
+}));
+
+const renderWarningIcon = (
```
Comment (pyphilia) [inline src/components/views/admin/KeywordsTable.tsx:34]: Why not be a real component?

File: src/components/common/table/ReadableTextField.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  value: string | unknown;
+  rowId: string;
+  fieldName: string;
+  onChange: (value: string) => void;
+};
+
+// Set the same padding as for the text input to avoid
+// movements between Typography and Textfield texts.
+const ReadyonlyTextField = styled(Typography)({
+  padding: '8px 14px',
```
Comment (pyphilia) [inline src/components/common/table/ReadableTextField.tsx:18]:
```suggestion
  padding: 'theme.spacing(1)px theme.spacing(2)px',
```
but with a correct syntax aha

**Review by spaenleh — APPROVED** [review]
Looks good, just some suggestions !

**Pull request merged**